### DISCUSSION TITLE
Any special handling required when migrating from graphql-upload for large files

### DISCUSSION THREAD

**Discussion by L-U-C-K-Y** [flash-oss/graphql-upload-minimal#35] [Q&A] [answered]
First of all, let me say that you are our life saver!

`graphql-upload` is ESM only and for the older packages, the `DefinitelyTyped` types got removed so our entire codebase was unable to work (https://github.com/DefinitelyTyped/DefinitelyTyped/pull/62785)
I added `graphql-upload-minimal` as a drop-in replacement and wanted to ask if there is anything special to check besides in the `README`.
I am asking because I read that nothing is written to disk, and was wondering if there is a concern with files that are 300MB+.

If anyone else has this type error:

```
error TS2688: Cannot find type definition file for 'fs-capacitor'.
   The file is in the program because:
     Entry point for implicit type library 'fs-capacitor'
 ```

Remove `graphql-upload` and use this library as a replacement.

Thank you!

**Comment by koresar** [comment, marked as answer]
As far I remember there is nothing else to check besides the README.md.
